frontend/app/controllers/Info.scala

```diff
@@ -66,7 +67,17 @@ trait Info extends Controller {
     )
   }
 
-  def supporterAustralia = CachedAndOutageProtected { implicit request =>
+  def supporterAustralia = NoCacheAction { implicit request =>
```

Comment:
Heavy traffic is relative. https://dashboard.ophan.co.uk/top20?host=membership.theguardian.com and https://dashboard.ophan.co.uk/info?host=membership.theguardian.com&path=/au/supporter suggest that we should be able to handle it.

Having said that, it's always good to cache if it doesn't add too much complexity.